runtests

```diff
@@ -138,6 +145,15 @@ if [[ -z "$CMDFILES" ]] && [[ -z "$CMDLINE" ]]; then
   err "no test to run!"
 fi
 
+START_TIME="$(date +%Y-%m-%d_%H-%M-%S)"
+SUMMRY_LOG="/tmp/lkvs_${START_TIME}_summary.log"
+echo "Test Start Time: $START_TIME" > "$SUMMRY_LOG"
+{
+  echo "--------------------------------------------------------"
+  echo "Testcase                                                                    Result     Exit Value  DURATION"
```

Comment:
Better to use `printf` here to make this line short

```shell
printf "%-76s%-11s%-12s%-8s" "Testcase" "Result" "Return Code" "Duration"
```

Reply:
Thanks for suggestion!
I will improve it.
Thanks!